src/markdown.c

```diff
@@ -1403,7 +1403,7 @@ prefix_quote(uint8_t *data, size_t size)
 	if (i < size && data[i] == ' ') i++;
 	if (i < size && data[i] == ' ') i++;
 
-	if (i < size && data[i] == '>') {
+	if ((i < size && data[i] == '>') && (i + 1 < size && data[i+1] != '!')) {
```

Comment:
I pretty much copied the blockquote logic, so blockspoilers should behave exactly as blockquotes